aep/general/0158/aep.md.j2

```diff
@@ -1,100 +1,103 @@
 # Pagination
 
-APIs often need to provide collections of data, most commonly in the
-[List][aip-132] standard method. However, collections can often be arbitrarily
-sized, and also often grow over time, increasing lookup time as well as the
-size of the responses being sent over the wire. Therefore, it is important that
-collections be paginated.
+APIs often need to provide collections of data, most commonly in the [List][]
+standard method. However, collections can often be arbitrarily sized, and also
+often grow over time, increasing lookup time as well as the size of the
+responses being sent over the wire. Therefore, it is important that collections
+be paginated.
 
 ## Guidance
 
-Operations returning collections of data **must** provide pagination _at the
-outset_, as it is a [backwards-incompatible change](#backwards-compatibility)
-to add pagination to an existing method.
+RPCs returning collections of data **must** provide pagination _at the outset_,
```

Comment:
```suggestion
Methods returning collections of data **must** provide pagination _at the outset_,
```

We've settled on "method" as the proper generic term for referring to things that may be gRPC or OAS (or something else).

Reply:
I thought it was operations? e.g. https://aep.dev/151. @mkistler - I forgot where we landed, but we're still mixing that nomenclature in AEPs.